api/services/account_service.py

```diff
@@ -698,3 +749,8 @@ def _get_invitation_by_token(cls, token: str, workspace_id: str, email: str) ->
 
             invitation = json.loads(data)
             return invitation
+
+
+def _generate_refresh_token(length: int = 64):
```

Comment:
```suggestion
def _generate_refresh_token(length: int = 64)->str:
```

Reply:
Considering that in this situation the LSP has sufficient type inference capabilities, I personally lean towards not adding this annotation.